src/harness/fake_sas.py

```diff
@@ -260,6 +260,9 @@ def TriggerDailyActivitiesImmediately(self):
   def GetDailyActivitiesStatus(self):
     return {'completed': True}
 
+  def InjectCpiUser(self, request):
```

Comment:
Actually you just need to add something near (what is currently) line 310. We only define the functions if we need to return a value to the test harness; in the cases where we don't need to return anything, we just add to the list below.